docs/01-create-twitter-app: Add note about read&write permissions (#133)
gr2m/twitter-together: docs/01-create-twitter-app.md

## Save credentials

> Your app must have "read and write" permissions. If you change the permission but have already generated user tokens, you have to re-generate them.

Reviewer: ```suggestion
"read and write" permissions are required. If you change the permission you must re-generate tokens for the change to take effect
```
